staging/src/k8s.io/apiserver/pkg/cel/builtins/compilation.go

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
Porting this file would be valuable for adding validation rules on native types (e.g. `// +validationRule: "<cel expression>")` but not for admission control, since we compile admission control quite differently. Are there plans to support validation rules on native types? If not it might be good to shelve this until then.

Reply:
All unused files removed.